### PR TITLE
Add a startup delay to the autoupdater,  to avoid interfering with critical work

### PR DESCRIPTION
**Pull request by directionless**
When launcher first enrolls with a server, it often begins fetching key data.  However, the autoupdater _also_ starts. And if there's a pending update, this has the effect of pre-empting that early data. 

This introduces a simple option to delay the autoupdater to allow key data to come in first. This option defaults to an hour.

### CODE REVIEW

File: cmd/launcher/updater.go [hunk 1]
```diff
@@ -64,6 +65,15 @@ func createUpdater(
 
 	return &actor.Actor{
 		Execute: func() error {
+			// When launcher first starts, we'd like the
+			// Saas side to start recieving data
```
Comment (blaedj) [inline cmd/launcher/updater.go:69]:
```suggestion
			// server to start receiving data
```

**Review by blaedj — APPROVED** [review]
lgtm, though I think we should note that this delay will happen every time launcher starts (reboot etc), right?

**Comment by directionless** [comment]
> lgtm, though I think we should note that this delay will happen every time launcher starts (reboot etc), right?

Yes, this delays applies on eery launcher start. Where do you think we should say that? I think it's implied by the option help text.

**Comment by blaedj** [comment]
> .. Where do you think we should say that? .. 

I just did, above 😄 the option help text is fine as-is, I agree.

**Pull request merged**